lib/ember-cli/configuration.rb

```diff
@@ -17,6 +17,12 @@ def tee_path
       @tee_path = Helpers.which("tee")
     end
 
+    def bower_path
+      @bower_path ||=
+        Helpers.which("bower") ||
+        Rails.root.join("node_modules", ".bin", "bower")
```

Comment:
Use 2 (not 0) spaces for indenting an expression in an assignment spanning multiple lines.